UnitsNet/GeneratedCode/Quantities/Pressure.g.cs

```diff
@@ -155,6 +155,11 @@ public Pressure(double value, UnitSystem unitSystem)
         /// </summary>
         public static Pressure MinValue { get; } = new Pressure(double.MinValue, BaseUnit);
 
+        /// <summary>
+        /// Represents the pressure at which Pressure is referenced (1 atm default)
+        /// </summary>
+        public static Pressure Reference { get; set; } = new Pressure(1, PressureUnit.Atmosphere);
```

Comment:
I don't think a static property should be settable. It can cause too much problems if 3rd party code start manipulating this value and expect this or that. Can it be made readonly getter property?

Reply:
Reviewing this property, it is not referenced anywhere, and was added to generated code. Considering `ReferencePressure` wraps `Pressure` to tie it to a pressure reference, I would expect implementation to access `ReferencePressure` and `Pressure` would be referenced for other uses of pressure, such as material stress, which unless I'm mistaken, does not require a reference pressure.
That said, I think this property can be removed as redundant.